docs/faqs/NuGet-FAQ.md

```diff
@@ -161,7 +161,7 @@ NuGet recommends all packages to be signed. A design principle of package signin
 
 Note that dependencies listed for your package are generated automatically from the package itself and cannot be edited.
 
-In addition, uploading packages to [int.nugettest.org](https://int.nugettest.org) is a great way to test and validate your package without making a package available in the public gallery.
+In addition, uploading packages to [int.nugettest.org](https://int.nugettest.org) is a great way to test and validate your package without making a package available in the public gallery. (EndPoint Uri is https://apiint.nugettest.org/v3/index.json)
```

Comment:
Instead of EndPoint, how about "Source"? That's the term we use on nuget.exe, dotnet.exe, and Visual Studio.

Reply:
The section we edited recently uses api endpoint.

https://docs.microsoft.com/en-us/nuget/faqs/nuget-faq#what-is-the-api-endpoint-for-nugetorg

following that...perhaps i should change EndPoint Uri to "API Endpoint"

Reply:
This doesn't need to be in parenthesis.

Also, "The API endpoint...". The P is not capitalized in endpoint.